extensions/arc/src/localizedConstants.ts

```diff
@@ -23,7 +23,8 @@ export const properties = localize('arc.properties', "Properties");
 export const settings = localize('arc.settings', "Settings");
 export const security = localize('arc.security', "Security");
 export const computeAndStorage = localize('arc.computeAndStorage', "Compute + Storage");
-export const nodeParameters = localize('arc.nodeParameters', "Node Parameters");
+export const coordinatorNodeParameters = localize('arc.coordinatorNodeParameters', "Coordinator Node Parameters");
+export const workerNodesParameters = localize('arc.workerNodesParameters', "Worker Nodes Parameters");
```

Comment:
"Worker Node Parameters" (singular node) would be consistent with what it's called in the PaaS portal.